frontend/src/components/annotation/annotation_ui_factory.vue

```diff
@@ -33,7 +33,7 @@
               :filtered_instance_type_list_function="filtered_instance_type_list"
               :show_default_navigation="show_default_navigation"
               :current_label_file="annotation_ui_context.current_label_file"
-              :has_changed="annotation_ui_context.get_current_ann_ctx() && annotation_ui_context.get_current_ann_ctx().has_changed || has_pending_frames"
+              :has_changed="has_changed"
```

Comment:
Created a proper reactive property `has_changed` and bound it to `annotation_ui_context.get_current_ann_ctx().has_changed`. Without it component didn't know when `has_change` was changing so `auido_toolbar` didn't enable "Save" button.